manim/mobject/svg/svg_mobject.py

```diff
@@ -124,28 +125,28 @@ def _ensure_valid_file(self):
         if self.file_name is None:
             raise Exception("Must specify file for SVGMobject")
 
-        if os.path.exists(self.file_name):
+        if Path(self.file_name).exists():
             self.file_path = self.file_name
             return
 
-        relative = os.path.join(os.getcwd(), self.file_name)
-        if os.path.exists(relative):
+        relative = Path.cwd().joinpath(self.file_name)
+        if Path(relative).exists():
```

Comment:
Instead of `Path(relative)` you can just use `relative` as that is already a path object.

Reply:
Fixed!